pkg/apis/api.acorn.io/v1/types.go

```diff
@@ -223,10 +223,11 @@ type ImageDetails struct {
 	NoDefaultRegistry bool `json:"noDefaultRegistry,omitempty"`
 
 	// Output Params
-	AppImage   v1.AppImage   `json:"appImage,omitempty"`
-	AppSpec    *v1.AppSpec   `json:"appSpec,omitempty"`
-	Params     *v1.ParamSpec `json:"params,omitempty"`
-	ParseError string        `json:"parseError,omitempty"`
+	AppImage        v1.AppImage   `json:"appImage,omitempty"`
+	AppSpec         *v1.AppSpec   `json:"appSpec,omitempty"`
+	Params          *v1.ParamSpec `json:"params,omitempty"`
+	SignatureDigest string        `json:"signatureDigest,omitempty"`
```

Comment:
Should this be apart of the `AppImage` struct such that I can reference `imageDetails.AppImage.SignatureDigest`?

Reply:
I think we aligned on treating signatures as separate entities completely, which are just referencing AppImages by Digest. At least @ibuildthecloud wanted to have them only coupled on the ImageDetails level.

Reply:
Ah okay. That does make sense. The only benefit that I saw is that some parts of the code become a bit simpler. However, if we want to treat them differently then I don't think that this is something we want to change.